sawtooth-sdk-transaction-processor/src/main/java/sawtooth/sdk/processor/TransactionProcessor.java

```diff
@@ -50,13 +53,13 @@
   private Stream stream;
 
   /** List of transaction handlers for this processor. */
-  private ArrayList<TransactionHandler> handlers;
+  private List<TransactionHandler> handlers;
```

Comment:
Based on the commit message, this change should be in  it's own commit.

Reply:
fair enough.  Slipped through.  Ends up getting replaced anyway in the next commit with a different type